ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -3592,6 +3592,8 @@ tests:
   steps:
     cluster_profile: vsphere-elastic
     env:
+      COMPUTE_ZONE: zone-1
```

Comment:
Shall we update ci job name to reflect the custom zones setting ?

Reply:
host-groups is one kind of zones, exsi host as zone

Reply:
So the customized zone is a MUST for the job ? or else, it will fail ?

Reply:
It's not a Must, do you mean change job name vsphere-ipi-host-groups-f2 to something like vsphere-ipi-host-groups-customize-f2?

Reply:
Yeah, that is my proposal, so that we can understand the test purpose for the ci jobs.

Reply:
Fixed, thanks.